build_debian.sh

```diff
@@ -67,6 +67,9 @@ mkdir -p $FILESYSTEM_ROOT/$PLATFORM_DIR
 mkdir -p $FILESYSTEM_ROOT/$PLATFORM_DIR/x86_64-grub
 touch $FILESYSTEM_ROOT/$PLATFORM_DIR/firsttime
 
+## ensure proc is mounted
+sudo mount proc /proc -t proc || true
```

Comment:
I see `mount proc` below. Do you want to move below code here?

Reply:
@qiluo-msft , do you mean line 100?  It is to mount /proc inside of the FILESYSTEM_ROOT, not directly for the slave container.